java/mtsj/core/pom.xml

```diff
@@ -129,7 +133,11 @@
     <dependency>
       <groupId>io.jsonwebtoken</groupId>
       <artifactId>jjwt</artifactId>
-      <version>0.7.0</version>
+    </dependency>
```

Comment:
shoundn't `jjwt` now come via devon4j jwt module and MTSJ only use this indirectly so this dependency is not needed anymore?